crypto/finite_field.ts

```diff
@@ -195,5 +195,12 @@ function createField(p: bigint, t: bigint, twoadicRoot: bigint) {
       }
       return BigInt('0b' + binary.reverse().join(''));
     },
+    leftShift(x: bigint, bits: number, maxBitSize: number = 64) {
+      let shifted = x << BigInt(bits);
+      return mod(BigInt('0b' + shifted.toString(2).slice(-maxBitSize)), p);
```

Comment:
not sure if doing the `mod` here is needed. To the reviewer, should this be removed?

Reply:
it wont do anything, assuming that `maxBitSize` is always `maxBitSize <= p`, but I start to agree with Gregor that these functions shouldnt belong in the `finite_field` library. I would remove `mod` and in a future PR clean up all the bitwise operations so that they dont introduce "impurities" in the ff lib :D